docs/operate/customize/requests.md

```diff
@@ -0,0 +1,72 @@
+# Requests
+
+_Introduced in InvenioRDM v14_
+
+Configure the requests system in InvenioRDM, including the reviewers feature for enhanced review workflows.
+
+## Reviewers Configuration
+
+The reviewers feature enables assignment of external experts, collaborators, or community members to provide feedback on requests without granting them decision-making authority.
+
+### Enable/Disable Reviewers
+
+```python
+# Enable the reviewers feature (default: True)
+REQUESTS_REVIEWERS_ENABLED = True
+```
+
+### Reviewer Limits
+
+Control the maximum number of reviewers that can be assigned to a single request. This helps manage workflow complexity and ensures review processes remain manageable.
+
+```python
+# Maximum number of reviewers per request (default: 10)
+REQUESTS_REVIEWERS_MAX_NUMBER = 5
+```
+
+- **Purpose**: Prevents overwhelming requests with too many reviewers
+- **Considerations**: Balance between comprehensive review and manageable workflow
+
+### Group Reviewers
+
+Enable the assignment of user groups as reviewers, allowing entire teams or committees to be assigned to review requests collectively.
+
+```python
+# Enable assignment of groups as reviewers (requires invenio-users-resources)
+USERS_RESOURCES_GROUPS_ENABLED = True
```

Comment:
I would remove this flag (from the code): if groups is enabled, why having this extra flag? Any benefit?

Reply:
this is the flag that enable groups in the system, it was not newly added.